packages/keystone/prepareKeystoneApp.js

```
@@ -4,16 +4,31 @@
 const conf = require('@open-condo/config')
 const { getRequestIp } = require('@open-condo/miniapp-utils/helpers/proxying')
 
+const { getLogger } = require('@open-condo/keystone/logging')
+
+const logger = getLogger('prepareKeystoneApp')
+
 async function prepareKeystoneExpressApp (entryPoint, { excludeApps } = {}) {
     debug('prepareKeystoneExpressApp(%s) excludeApps=%j cwd=%s', entryPoint, excludeApps, process.cwd())
+    logger.info({msg: 'start prepare keystoneExpressApp', data: { entryPoint, excludeApps, processCwd: process.cwd() }})
```

ESLint

[error] A space is required after '{'.

(@typescript-eslint/object-curly-spacing)